index.html

```diff
@@ -17,6 +17,17 @@
         latestVersion: "https://swicg.github.io/activitypub-data-portability/",
         shortName: "data-portability-report",
         group: "socialcg",
+        otherLinks: [
```

Comment:
See https://respec.org/docs/#otherLinks for documentation on this reSpec property